### DISCUSSION TITLE
Hi, how to send message to group?

### DISCUSSION THREAD

**Discussion by ariaseta** [aldinokemal/go-whatsapp-web-multidevice#124] [Q&A] [answered]
<img width="1118" alt="image" src="https://github.com/aldinokemal/go-whatsapp-web-multidevice/assets/1048940/f49a43a4-5fb9-4b24-957c-3995c724d402">

I found error when using group id

**Comment by aldinokemal** [comment, marked as answer]
currently I see in your image still using type *Private Message*

if group you have to change type become group
so it will end with @g.us instead of @s.whatsapp.net
